config/routes.rb

```diff
@@ -0,0 +1,108 @@
+Rails.application.routes.draw do
```

Comment:
It looks like you are using a lot of standard routes. It might make more sense to use resources rather than define each of the specific routes out as you have done.